app/master/cluster_master.py

```diff
@@ -296,7 +297,7 @@ def handle_result_reported_from_slave(self, slave_url, build_id, subjob_id, payl
         :rtype: str
         """
         self._logger.info('Results received from {} for subjob. (Build {}, Subjob {})', slave_url, build_id, subjob_id)
-        build = BuildStore.get(int(build_id))
+        build = self._build_store.get(int(build_id))
```

Comment:
Nice catch, looks like we convert the `build_id` to an int before we ever pass it into this function.

e.g.

```python
self._cluster_master.handle_result_reported_from_slave(
            slave_url, int(build_id), int(subjob_id), file_payload[0])
```